inject/inject.vala

```diff
@@ -13,6 +14,7 @@ namespace Frida.Inject {
 		{ "pid", 'p', 0, OptionArg.INT, ref target_pid, null, "PID" },
 		{ "name", 'n', 0, OptionArg.STRING, ref target_name, null, "PID" },
 		{ "script", 's', 0, OptionArg.FILENAME, ref script_path, null, "JAVASCRIPT_FILENAME" },
+		{ "spawn", 'f', 0, OptionArg.NONE, ref spawn, "Spawn target application", null },
```

Comment:
This should be a string, not a boolean, and should be `--file` in long-form, like in frida-tools:

`-f FILE, --file=FILE  spawn FILE`

E.g. `frida-inject -f /bin/cat /etc/hosts`.

Extra arguments should be collected as the arguments to pass.